documentation/decisions/adr/000n-design-diagramming-tool.md

```diff
@@ -0,0 +1,118 @@
+# Use Mural for design diagrams and whiteboarding
+
+- **Status:** Proposed <!-- REQUIRED -->
+- **Last Modified:** {YYYY-MM-DD when the decision was last updated} <!-- REQUIRED -->
+- **Related Issue:** [#116](https://github.com/HHS/grants-api/issues/116) <!-- RECOMMENDED -->
+- **Deciders:** Andy Cochran, Emily Ianacone, Sumi Thaiveettil, Aaron Couch, Billy Daly, Lucan Brown <!-- REQUIRED -->
+- **Tags:** design <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+Which tool should be used for diagramming and whiteboarding? This tool is a primary concern for design in the creation of low-fidelity wireframes, process/logic flows, journey maps, service blueprints.
+
+This diagramming tool will not be used only by designers. The right option will double as a whiteboarding tool for all disciplines, eliminating the need for a separate whiteboard tool. However, the goal here is not to prevent duplicative tools. There are programmatic methods of generating diagrams (e.g. MermaidJS) that will likely be used for creating architectural diagrams and tecnhical documentation in a machine-readable, version-controled format.
+
+Note that the intended use for this particular tool is collaborative drawing.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- Collborative editing
+- Functions as both a digramming tool and a general whiteboard
+- Useful to all disciplines (design, engineering, product, project)
+
+## Options Considered
+
+- Mural
+- Lucidchart
+- Miro
+- Visio
+- Figjam
+- Draw.io
+- ClickUp
+
+## Decision Outcome <!-- REQUIRED -->
+
+Chosen option: Mural, because HHS has existing licenses and Nava has extensive experience using it for multiple purposes (diagrams, whiteboard, research synthesis, brainstorming, etc.). Mural is very effective as a collaborative drawing canvas. And it will be valuable to all disciplines for various purposes.
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- Mural can be used for sprint retrospective board, eliminating the need for an retro-specific tool
```

Comment:
This is a helpful observation! We do have "Sprint Retro Tool" listed as one of the ADRs associated with the Comms Platforms milestone. Mural should at the very least be one of the options we consider, if not the one we select just to get things rolling this sprint.